pkg/v6/configmap/create_test.go

```diff
@@ -155,7 +155,7 @@ func Test_ConfigMap_newCreateChange(t *testing.T) {
 	}
 
 	c := Config{
-		Guest:          &guestMock{},
+		Tenant:         &guestMock{},
```

Comment:
The test changes I kept minimal. From `v7` onwards the tests use the correct naming.